src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -633,6 +633,7 @@ private function registerWorkflowConfiguration(array $workflows, ContainerBuilde
                     new Reference('security.authorization_checker'),
                     new Reference('security.authentication.trust_resolver'),
                     new Reference('security.role_hierarchy'),
+                    new Reference('validator'),
```

Comment:
Ok, I will add : 

```php
new Reference('validator', ContainerInterface::NULL_ON_INVALID_REFERENCE),
```

Should I also add
```php
new Reference('security.role_hierarchy', ContainerInterface::NULL_ON_INVALID_REFERENCE),
```

As `$roleHierarchy` parameter in `Symfony\Component\Workflow\EventListener\GuardListener` can be null too ?